pkg/agent/route/route_windows.go

```diff
@@ -129,10 +135,135 @@ func (c *Client) listRoutes() (map[string]*netroute.Route, error) {
 		if rt.LinkIndex != c.nodeConfig.GatewayConfig.LinkIndex {
 			continue
 		}
-		if rt.DestinationSubnet.IP.IsLoopback() {
+		// Only process IPv4 route entries in the loop.
+		if rt.DestinationSubnet.IP.To4() == nil {
 			continue
 		}
-		rtMap[rt.DestinationSubnet.String()] = &rt
+		// Retrieve the route entries with destination using global unicast only. This is because the function
+		// "GetNetRoutesAll" also returns the entries of loopback, broadcast, and multicast, which are
+		// added by the system when adding a new IP on the interface. Since removing those route entries might
+		// introduce the host networking issues, ignore them from the list.
```

Comment:
would there be problem if we don't clean these broadcast and multicast routes?

Reply:
I don't think there is problem if we don't remove such routes. But if we remove them, the host might not ping gw0.

These routes are added by windows automatically. I think it should be safe to keep them.

Reply:
So these routes are not added when adding a remote Node?

Reply:
No, only one route is added each time a new Node is joining the cluster. These routes are added when new IP is configured on the host, or I should say when new IP is configured on gw0.